crates/subspace-networking/src/create.rs

```diff
@@ -86,66 +81,7 @@ const REGULAR_BASE_CONCURRENT_TASKS: usize = 120 - KADEMLIA_BASE_CONCURRENT_TASK
 /// Above base limit will be boosted by specified number for every peer connected starting with
 /// second peer, such that it scaled with network connectivity, but the exact coefficient might need
 /// to be tweaked in the future.
-const REGULAR_CONCURRENT_TASKS_BOOST_PER_PEER: usize = 2;
-/// How many peers should node be connected to before boosting turns on.
-///
-/// 1 means boosting starts with second peer.
-const CONCURRENT_TASKS_BOOST_PEERS_THRESHOLD: NonZeroUsize =
```

Comment:
You deleted this constant, but I'd really like to keep it. It is important to not exceed limits and this threshold provides us a buffer to boost more safely.